### ISSUE TITLE
[Feature request] Implement SSO

### ISSUE THREAD

**Issue by sideeffectdk** [YunoHost-Apps/planka_ynh#9]
Heya contributers.
Nice work on getting Planka in yunohost. I'm already loving it immensely and getting people on (kanban)board with it.

To help ease the process of onboarding new users, it would be a great help to get the standard SSO/LDAP integrated, so we don't have to create new users and log in seperately when visiting the kanban board.

I would love for SSO to be implemented, but I'm no genius in this matter and not sure it's even possible in the current state of both Planka and yunohost, and/or if the SSO methods are compatible. I've seen mentions of Authentik, Authelia, OIDC, all other SSO methods than SSOWat.

Here is where I have searched for answers:
https://docs.planka.cloud/docs/Configuration/OIDC
https://github.com/plankanban/planka/issues?q=sso
https://github.com/plankanban/planka/pulls?q=sso

Thanks. Have a great day!
- sideeffectdk

**Comment by Limezy** [comment]
The Dex app should be used for that. I'm currently contemplating switching my kanban from Wekan to Planka.
If I do so, I'll definitely try to implement a Dex-based OIDC version of Planka, to leverage the sso.

**Comment by CtheCondor** [comment]
Planka using the OIDC connection works great with Authentik so that is another SSO provider like DEX that does work. Planka's documentation in particular showcases sample configuration with Authentik.

**Comment by grenagit** [comment]
> The Dex app should be used for that. I'm currently contemplating switching my kanban from Wekan to Planka.
> If I do so, I'll definitely try to implement a Dex-based OIDC version of Planka, to leverage the sso.

Yes, indeed, Dex would have been the solution to implement SSO via OIDC.

Unfortunately, [Planka Community 2.2.0](https://github.com/plankanban/planka/releases/tag/v2.2.0) has removed OIDC/SSO...
